integration_tests/sdk/local_test.py

```diff
@@ -0,0 +1,51 @@
+from doctest import OutputChecker
+from pandas import DataFrame
+import pytest
+import aqueduct
+from test_metrics.constant.model import constant_metric
+from test_functions.simple.model import dummy_sentiment_model, dummy_sentiment_model_multiple_input
+from test_functions.sentiment.model import sentiment_model
+from checks_test import success_on_single_table_input
+from constants import SENTIMENT_SQL_QUERY
+from utils import (
+    get_integration_name,
+    run_sentiment_model,
+    should_run_complex_models
+)
+
+def test_local_operator(sp_client):
+    db = sp_client.integration(name=get_integration_name())
+    sql_artifact = db.sql(query=SENTIMENT_SQL_QUERY)
+    output_artifact = run_sentiment_model(sql_artifact)
+    output_cloud = output_artifact.get()
+    output_local = sentiment_model.local(sql_artifact) if should_run_complex_models() else dummy_sentiment_model.local(sql_artifact)
+    assert output_cloud.count()[0] == output_local.count()[0]
+    assert (output_cloud["positivity"] == output_local["positivity"]).all()
```

Comment:
is there a reason we can't compare the entire dataframe for equality? They should be exactly the same right?

Reply:
Yes I can check the entire Dataframe for equity. my initial thought is that the testing operation only add a column "positivity" to the Dataframe but it will be good to check if other columns are affected or not.